code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -19,6 +19,7 @@
 
 /obj/machinery/atmospherics/components/update_icon()
 	update_icon_nopipes()
+	update_icon_state()
 
 	underlays.Cut()
```

Comment:
Add return ..() on line 51 and line 34 instead thats should fix it better

(All update icons should be calling parent but because we have so much broken code not doing it we can't lint it yet)

Reply:
Okay done